### DISCUSSION TITLE
Mapping a new narrative for north west lutruwita

### DISCUSSION THREAD

**Discussion by lianaguegan** [life-itself/community#1088] [General]
Life Itself lutruwita in unison with The Reseed Centre (a small sustainability centre in north west lutruwita/Tasmania) has launched a mapping project beneath the inspirational light of mapping efforts at Life Itself in Europe and other projects across the planet. The ‘accessible to all’ map hopes to raise visibility of emergent culture in the north west. The idea is that initiatives can become aware of others in the ecosystem for the purpose of networking, collaboration and widening broadcasting scope, and can at the same time achieve heightened visibility in the broader north west lutruwita community. It is envisaged that local governments, schools, healthcare bodies, tourist information, churches, clubs, individuals etc can have access to the map as a resource to link into for growing regional wellness and thriving. The map is using the 3 permaculture ethics, Earth Care, People Care and Fair Share as its emergent culture container, and initiatives will be mapped onto the the Permaculture Petal (Land and Nature Stewardship, Health and Spiritual Wellbeing, Education and Culture, Land Tenure and Community Governance, Built Environment, Finance and Economics, and Tools and Technology).

In part inspiration for the map has come from The Charter of Compassion’s reasons for mapping their community: ‘we have the potential for exponentially more impact by working together’. The premise of Radical Cross-Promotion which steps away from a culture of competition and individualism to one of care and collaboration is functioning as a primary building block. In this model, mapping is the first step in encouraging groups to connect and align with others to incorporate activity and event details into their respective communication networks, and so increase networking and public visibility. It aims to open up ‘tribe’ finding capacity across the ecosystem despite which group is doing the marketing. Most importantly the map hopes to rewrite the narrative of north west lutruwita as a locale of life affirming culture where dynamic networks of thriving reside.

**Comment by rufuspollock** [comment]
@lianaguegan great - and what would support you in the mapping project?

**Comment by lianaguegan** [comment]
Thank you Rufus 🙏🏼 I’m just touching base with the team. I will see what they say.Warmest, liana The current state of the planet has arisen from the thoughts and actions of humans past and present. The ethic of peoplecare includes changing the thinking behind our actions. Once our thinking changes our actions will follow suit (Looby Macnamara)Wild Plum understands that the ancient soils of lutruwita belong to the palawa people, apologises for invasion, and works these soils with gratitudeOn 2 Jun 2024, at 8:35 AM, Rufus Pollock ***@***.***> wrote:﻿
@lianaguegan great - and what would support you in the mapping project?
